example/package.json

```diff
@@ -11,6 +11,7 @@
     "sync": "/usr/bin/rsync -v -a ../src ./node_modules/react-native-week-view/"
   },
   "dependencies": {
+    "memoize-one": "^5.1.1",
```

Comment:
this dependency should be from the library, not from the example project.

Reply:
whoops, you're right. will fix ASAP

Reply:
Updated